docker-compose.yml

```diff
@@ -17,7 +21,7 @@ services:
       SSH_HOST: ssh-tunnel
       SSH_PORT: 22
       SSH_USER: root
-    links:
+    depends_on:
```

Comment:
- ssh_tunel sa musi vytvorit skor, kedze obsahuje zdielane SSH kluce, ... neviem preco to bez toho islo, mozno nejaka zmena v Docker.